components/03-sections/01-accordion/accordion.hbs

```diff
@@ -22,15 +29,58 @@
                     </div>
                 </div>
             </div>
-            <div class="card">
-                <div class="card-header"> <a class="card-link" data-bs-toggle="collapse"
-                        href="#collapse-02">{{ heading }}</a>
+            <div class="accordion-item">
+                <h5 class="accordion-header" id="heading-02">
+                    <button type="button" class="accordion-button collapsed" data-bs-toggle="collapse"
+                        data-bs-target="#collapse-02">
+                        {{heading}}
+                    </button>
+                </h5>
+                <div id="collapse-02" class="accordion-collapse collapse" data-bs-parent="#accordion">
+                    <div class="card-body">
+                        <div class="accordion-body-content">
+                            <h5>
+                                {{title}}
+                            </h5>
+                            <p>
+                                {{text}}
+                            </p>
+                            <ul class="ul-list">
+                                <li class="list-item">Use the HTML <ol> element to define an ordered list</li>
+                                <li class="list-item">Use the HTML type attribute to define the numbering type</li>
+                                <li class="list-item">Use the HTML
+                                <li> element to define a list item</li>
```

Comment:
Please add backticks (``) around `<li>` element, so it doesn't cause a code break.